### PR TITLE
Task 65 - User DTO modification

### PR DESCRIPTION
**Pull request by RobertoJavaDev**
We need to think about improving/enhancing these tests so that we do not create such test objects:
UserRequestDto userRequestDto = new UserRequestDto(
                "Łamignat",
                "lamignar@gmail.com",
                "password",
                null,
                null,
                null,
                null);

### CODE REVIEW

File: src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java [hunk 1]
```diff
@@ -91,13 +104,22 @@ void shouldGetSingleUser() {
     @Rollback
     void shouldCreateUser() {
         // given
-        UserRequestDto userRequestDto = new UserRequestDto("Łamignat", null);
+        UserRequestDto userRequestDto = new UserRequestDto(
+                "Łamignat",
+                "lamignar@gmail.com",
```
Comment (Manes79) [inline src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java:109]: lamignat@gmail.com? :)

File: src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java [hunk 2]
```diff
@@ -111,13 +133,22 @@ void shouldUpdateUser() {
         quizRepository.save(quiz);
         List<Quiz> quizzesList = new ArrayList<>();
         quizzesList.add(quiz);
-        UserRequestDto userRequestDto = new UserRequestDto("Gniewosz", quizzesList);
+        UserRequestDto userRequestDto = new UserRequestDto(
+                "Gniewosz",
+                "gniewoszr@gmail.com",
```
Comment (Manes79) [inline src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java:138]: gniewosz@gmail.com ? :)

**Review by Manes79 — APPROVED** [review]
probably small typos by the emails, the rest very good :)

File: src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java (hunk 1 above)
Comment (RobertoJavaDev) [reply src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java]: Done

File: src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java (hunk 2 above)
Comment (RobertoJavaDev) [reply src/test/java/pl/stormit/eduquiz/quizcreator/domain/user/UserServiceTest.java]: Done

**Pull request merged**